test/e2e/tests/request-queuing/multiple-networks-dapps-txs.spec.js

```diff
@@ -0,0 +1,172 @@
+const FixtureBuilder = require('../../fixture-builder');
+const {
+  withFixtures,
+  openDapp,
+  unlockWallet,
+  DAPP_URL,
+  DAPP_ONE_URL,
+  regularDelayMs,
+  WINDOW_TITLES,
+  defaultGanacheOptions,
+} = require('../../helpers');
+const { PAGES } = require('../../webdriver/driver');
+
+// TODO: Have to turn on the setting every time we want to test the setting!?!
```

Comment:
We should be able to add this to the fixture no?

Reply:
Yes, we should be able to. The issuesis that any type of refresh of the state that happens reverts the request queue setting behavior to default, the setting is off even though it is supposed to be on.